apply.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Place Order</title>
+    <link rel="stylesheet" href="src/apply.css">
+</head>
+<body>
+    <header>
+        <h1 class="order-header">Place your order</h1>
+    </header>
+
+    <main>
+        <form id="place-order">
+            <fieldset id="customer-fieldset">
+                <legend>
+                    Customer Details
+                </legend>
+                
+                <label for="name">
+                    Your name:
+                    <input type="text" name="name" id="name" placeholder="Enter your name..">
+                </label>
+
+                <br />
+
+                <label for="phone-number">
+                    Phone #:
+                    <input type="tel" name="phone-number" id="phone-number">
+                </label>
+
+                <br />
+                
+                <div>
+                    <p>return customer?</p>
+                    <label for="Yes">Yes</label>
+                    <input id="Yes" name="return-customer" type="radio" value="Yes">
+                
+                    <label for="No">No</label>
+                    <input id="No" name="return-customer" type="radio" value="No">
+                </div>
+            </fieldset>
+
+            <fieldset id="product-details">
+                <legend>
+                    Product Details
+                </legend>
+
+                <div>
+                    <label for="item-number">Item #</label>
+                    <input type="number" name="item-number">
+                </div>
+
+                <div>
+                    <label for="item-description">Item Description</label>
+                </div>
+
+                <div>
+                    <textarea id="text-box" name="item-description"></textarea>
+                </div>
+
+                <div>
+                    <label for="price">Unit Price $</label>
+                </div>
+
+                <div>
+                    <input type="number" name="price">
+                </div>
+            </fieldset>
+
+            <section>
+                <h2>
+                    Moment of Truth
+                </h2>
+
+                <button>Submit</button>
+            </section>
+
+        </form>
+    </main>
+    <script type="module" src="src/apply.js"></script>
+</body>
+</html>
+            
```

Comment:
Lots of unneeded white space at the bottom of this file. I would delete most of it if I were you.